medmodels/statistic_evaluations/comparer/data_comparer.pyi

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import polars as pl
+
+from medmodels.medrecord.medrecord import MedRecord
+from medmodels.medrecord.querying import NodeOperation
+from medmodels.medrecord.types import (
+    Group,
+    GroupInputList,
+    MedRecordAttribute,
+    MedRecordAttributeInputList,
+)
+from medmodels.statistic_evaluations.comparer.builder import ComparerBuilder
+
+class DataComparer:
+    patients_group: Group
+    time_attribute: MedRecordAttribute
+    concept_groups: GroupInputList
+    top_k_concepts: int
+    alpha: float
+    subcohorts_groups: Optional[GroupInputList]
+    subcohorts_queries: Optional[List[NodeOperation]]
+    control_cohort: Optional[Union[Group, NodeOperation]]
+
+    @classmethod
+    def builder(cls) -> ComparerBuilder: ...
+    @staticmethod
+    def _set_configuration(
+        data_comparer: DataComparer,
+        *,
+        patient_group: Group = "patients",
+        time_attribute: MedRecordAttribute = "time",
+        concept_groups: GroupInputList = ["concepts"],
+        top_k_concepts: int = 10,
+        alpha: float = 0.05,
+        subcohorts_groups: Optional[GroupInputList],
+        subcohorts_queries: Optional[List[NodeOperation]],
+        control_cohorts: Optional[Union[Group, NodeOperation]],
+    ) -> None: ...
+    def compare_attributes(
+        self,
+        medrecords: List[MedRecord],
+        attributes: MedRecordAttributeInputList,
+    ) -> Dict[MedRecordAttribute, pl.DataFrame]: ...
+    def calculate_absolute_relative_difference(
+        self,
+        medrecords: Union[MedRecord, List[MedRecord]],
+        attributes: MedRecordAttributeInputList,
+        control_data: Optional[MedRecord],
+    ) -> Tuple[float, pl.DataFrame]: ...
+    def test_difference_attributes(
+        self,
+        medrecords: Union[MedRecord, List[MedRecord]],
+        attributes: MedRecordAttributeInputList,
+    ) -> Dict[MedRecordAttribute, Dict[str, Union[str, bool, float]]]: ...
+    def get_top_k_concepts(
```

Comment:
Maybe it would be nice to be able to give top_k also here? So you do not have to modify the comparer to change the top_k?